.github/workflows/jekyll-gh-pages.yml

```diff
@@ -0,0 +1,51 @@
+# Workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["edge"]
```

Comment:
There have been some bug reports relating to the documentation being too up to date, in a manner of speaking. I'm not sure if it's really a problem though.